lib/pry/config/behavior.rb

```diff
@@ -14,13 +14,17 @@ module Behavior
 
       module Builder
         #
-        # Returns a new Behavior, non-recursively (unlike {#from_hash}).
+        # @example
+        #   c = Pry::Config.assign(foo: {bar: {baz: 42}})
+        #   c.class # => Pry::Config
+        #   c.foo.class # => Hash
         #
         # @param
         #   (see #from_hash)
         #
-        # @return
-        #   (see #from_hash)
+        # @return [Pry::Config::Behavior]
+        #   Returns an initialized object who included {Pry::Config::Behavior},
```

Comment:
To save space YARD assumes that `@return` already says "Returns", so this becomes:

` @return [Pry::Config::Behavior] an initialized object who included...`

Point is no need to write "return" twice.

Reply:
sure thanks, i'll update. i'm not that happy with these docs yet, thats why i marked as a wip for now.

Reply:
i updated, the only issue i had was with boolean returns, eg:
```
# @return [Boolean] ..?
```
how should that be written without using `returns` ?

Reply:
`true if X, false otherwise`

https://github.com/pry/pry/blob/dc8aacb26b43d07cc406d07cc4b802c972f11957/lib/pry/pry_instance.rb#L407

Reply:
done here: https://github.com/pry/pry/pull/1892/commits/8ef87eae74ba4aa595051de1b0f959b9a84773a5